apps/explorer-e2e/src/support/pages/blocks-page.ts

```diff
@@ -85,6 +83,37 @@ export default class BlocksPage extends BasePage {
       });
   }
 
+  navigateToLastBlockOnPage() {
+    this.waitForBlocksResponse();
+    cy.getByTestId(this.infiniteScrollWrapper).children().scrollTo('bottom');
+  }
+
+  navigateToOlderBlocksWithInfiniteScroll(
+    expectedBlocks: number,
+    scrollAttempts: number
+  ) {
+    cy.getByTestId(this.blockHeight)
+      .first()
+      .invoke('text')
+      .then(($firstBlockHeight) => {
+        for (let index = 0; index < scrollAttempts; index++) {
+          // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
question (blocking): Is this necessary? This will disable it for line 101, but the wait is on 104?

Reply:
So i found the wait to be essential to enable scrollTo to function correctly, i think it mimicks user interaction, the time taken for ones finger to re-interact with the scroll wheel between each scroll. 

The linter has auto-placed the comment, but that is just a comment right? I'm a little new to linting post commit